drivers/hmis/spec/requests/hmis/ce/ce_opportunity_spec.rb

```diff
@@ -275,7 +275,7 @@
     end
 
     context 'when the opportunity has some candidates the current user lacks permission to view' do
-      let(:candidate_pool_with_anonymous) { create :hmis_ce_match_candidate_pool }
+      let(:candidate_pool_with_anonymous) { create :hmis_ce_match_candidate_pool, priority_expression: '1' } # overwrite priority expression with a different dummy value to avoid duplicate key
```

Comment:
Can we use a `sequence` in the hmis_ce_match_candidate_pool factory instead? It would be preferable if each time you create a new instance using the factory, it is valid.